### PR TITLE
Add accessibility considerations regarding real-time text

### PR DESCRIPTION
**Pull request by dontcallmedom**
Based on discussions with APA Working Group during TPAC https://www.w3.org/2019/09/20-apa-minutes.html#item02 and following discussions

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/pull/2328.html" title="Last updated on Dec 2, 2019, 4:40 PM UTC (0e61aec)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2328/8738565...0e61aec.html" title="Last updated on Dec 2, 2019, 4:40 PM UTC (0e61aec)">Diff</a>

### CODE REVIEW

**Comment by dontcallmedom** [comment]
cc @brewerj @JaninaSajka @joshueoconnor

File: webrtc.html [hunk 1]
```diff
@@ -12928,6 +12928,51 @@ <h2>Setting SDP from remote endpoints</h2>
       SDP.</p>
     </section>
   </section>
+  <section class="informative">
+    <h2>Accessibility Considerations</h2>
+    <p>The WebRTC 1.0 specification exposes an API to control protocols
+      (defined within the IETF) necessary to establish real-time
+      audio, video and data exchange.</p>
+
+    <p>Real-time Text, defined in [[RFC4103]] is used by people who are hearing
```
Comment (alvestrand) [inline webrtc.html]: We can't be this positive about it. RFC 4103 is being obsoleted (as later text says), so we shouldn't write of it as if it was "up and coming".

Comment (aboba) [reply webrtc.html]: How about this alternative paragraph?

"The Telecommunications Device for the Deaf (TDD/TTY) enables
      individuals who are hearing or speech impaired (among others)
      to communicate over telephone lines. Real-time Text, defined
      in [[RFC4103]], utilizes T.140 encapsulated in RTP to enable
      the transition from TDD/TTY devices to IP-based communications,
      including emergency communication with
      <a href="https://www.ecfr.gov/cgi-bin/text-idx?SID=ada7cebd65433a9e05c35426a2bc76b8&amp;mc=true&amp;node=pt47.3.67&amp;rgn=div5">Public Safety Access Points (PSAP)</a>"

**Comment by alvestrand** [comment]
The text is not wrong in itself, but it seems completely out of place in an API definition spec.
It would fit very nicely into the webrtc-nv-use-cases document.

If organizational peace depends on having this text, I'm not going to stand in the way, though. It's "mostly harmless".

**Comment by dontcallmedom** [comment]
@JaninaSajka @joshueoconnor can you check the final text (which was changed a bit from the previously discussed draft) still works for APA?

**Comment by henbos** [comment]
Ping!

**Comment by dontcallmedom** [comment]
Having not heard back, I think we're fine merging the proposed text

**Comment by RealJoshue108** [comment]
Sorry @dontcallmedom and all for delay - I missed this request.

Are you asking about the suggested text:
""The Telecommunications Device for the Deaf (TDD/TTY) enables
individuals who are hearing or speech impaired (among others)
to communicate over telephone lines. Real-time Text, defined
in [[RFC4103]], utilizes T.140 encapsulated in RTP to enable
the transition from TDD/TTY devices to IP-based communications,
including emergency communication with
Public Safety Access Points (PSAPs)"??

Please confirm, and I will mention it to Janina today - thanks

**Comment by RealJoshue108** [comment]
I'm using a new GH account so for future reference please use @RealJoshue108 and I see the ping, thanks.

**Comment by RealJoshue108** [comment]
> The text is not wrong in itself, but it seems completely out of place in an API definition spec.
> It would fit very nicely into the webrtc-nv-use-cases document.

I also think for future work the WebRTC NV Use Cases type documents would be better for User Needs etc - there is a need to be explicit about WebRTC and support for RTT in the API doc, and we are happy to see text that reinforces this.

**Comment by joshueoconnor** [comment]
@domenic Just discussed this with @JaninaSajka and we have a wording concern about the phrase 'would enable' as it is predicting a result that we cannot be fully certain of and may be better stated as 'is expected to enable'.

**Comment by joshueoconnor** [comment]
Thanks for edit @dontcallmedom

**Pull request merged**